Parsers/ASimDns/Product parsers/ASimDnsCiscoUmbrella.yaml

```diff
@@ -27,7 +27,7 @@ ParserQuery: |
       // ******************* Mandatory
         Type='Cisco_Umbrella_dns_CL',
          EventCount=int(1),
-         EventStartTime=todatetime(Timestamp_t),
+         EventStartTime= column_ifexists("Timestamp_t", todatetime(column_ifexists("Timestamp_s",""))),
```

Comment:
This might be an issue with any field :-(

Reply:
true.  :( but this is the only place I actually saw this occurs